src/executables/master.c

```diff
@@ -436,6 +436,7 @@ css_accept_old_request (CSS_CONN_ENTRY * conn, unsigned short rid, SOCKET_QUEUE_
 static int
 receive_server_info (CSS_CONN_ENTRY * conn, unsigned short rid, std::string & dbname, SERVER_TYPE & type)
 {
+  // *INDENT-OFF*
```

Comment:
Extend the disabled indent code section to include function arguments (because there is an std::string argument).
```suggestion
// *INDENT-OFF*
static int
receive_server_info (CSS_CONN_ENTRY * conn, unsigned short rid, std::string & dbname, SERVER_TYPE & type)
{
```